doc/build-unix.md

```diff
@@ -197,17 +197,21 @@ Berkeley DB
 -----------
 
 The legacy wallet uses Berkeley DB. To ensure backwards compatibility it is
-recommended to use Berkeley DB 4.8. If you have to build it yourself, you can
-use [the installation script included in contrib/](/contrib/install_db4.sh)
-like so:
-
-```shell
-./contrib/install_db4.sh `pwd`
+recommended to use Berkeley DB 4.8. If you have to build it yourself, and don't
```

Comment:
Wondering if this section should be titled `Legacy Wallet Support` rather than `Berkeley DB`?